test/e2e/Makefile

```diff
@@ -42,9 +42,10 @@ e2e-cli-plugin-compatibility-test:
 	@if [ "${TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL}" = "" ]; then \
 		echo "***Skipping Plugin Compatibility test cases because environment variables TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL is not set***" ; \
 	else \
-		export TANZU_CLI_PRE_RELEASE_REPO_IMAGE=$(TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL) ; \
+		export TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL=$(TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL) ; \
 		export TANZU_CLI_PLUGIN_DISCOVERY_IMAGE_SIGNATURE_VERIFICATION_SKIP_LIST=$(TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL) ; \
 		export TANZU_CLI_CEIP_OPT_IN_PROMPT_ANSWER="Yes" ; \
+		export ALLOWED_REGISTRY=$$(echo ${TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL} | cut -f1 -d/ | cut -f1 -d:) ; \
```

Comment:
Why do we need to set this environment variable?

Reply:
The trusted registry list used to automatically include whatever registry was added in `TANZU_CLI_PRE_RELEASE_REPO_IMAGE`.  Now that we use plugin sources instead, I didn't think we should automatically trust whatever registry was added as a plugin source (or else it seems to defeat the purpose of the `ALLOWED_REGISTRY` variable).

So, for the E2E tests, which use a different registry than `projects.registry.vmware.com` we need to set the variable.

Makes sense?

Reply:
> The trusted registry list used to automatically include whatever registry was added in TANZU_CLI_PRE_RELEASE_REPO_IMAGE. Now that we use plugin sources instead, I didn't think we should automatically trust whatever registry was added as a plugin source (or else it seems to defeat the purpose of the ALLOWED_REGISTRY variable).

Why not trust the registry, if the registry was added by the user with the `tanzu plugin source update` command? 
I feel that we should NOT ask users to configure any additional variable except updating the discovery source. Also `ALLOWED_REGISTRY` was added as a backup and we haven't documented this to be used by end users in production even for air-gapped.